tests/algorithms/test_adversarial.py

```diff
@@ -433,3 +433,35 @@ def test_compute_train_stats(n_samples):
     for k, v in stats.items():
         assert isinstance(k, str)
         assert isinstance(v, float)
+
+
+def test_regression_gail_with_sac(pendulum_expert_trajectories, pendulum_venv):
+    """Right now GAIL with a SAC learner on GPU crashes when training (see #655).
+
+    This is a minimal test to reproduce it.
+
+    Args:
+        pendulum_expert_trajectories: expert trajectories for Pendulum env.
+        pendulum_venv: the Pendulum environment.
+    """
+    from stable_baselines3 import SAC
```

Comment:
Is there a reason we need to import these inside the test? We generally avoid that in this codebase and https://google.github.io/styleguide/pyguide.html#313-imports-formatting requires that "Imports are always put at the top of the file"